lib/src/ui/movie_detail.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:animate_do/animate_do.dart';
+import 'package:flutter/material.dart';
+import '../widgets/movie_detail_poster.dart';
+import '../widgets/movie_detail_subtitle.dart';
+import '../widgets/movie_detail_title.dart';
+import '../widgets/movie_detail_overview.dart';
+import '../widgets/movie_vote_average.dart';
+import '../utils/movie_dimensions.dart';
+import '../models/result_movie.dart';
+
+class MovieDetail extends StatelessWidget {
+  final ResultMovie result;
+
+  const MovieDetail({
+    Key? key,
+    required this.result,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SingleChildScrollView(
+        child: ZoomIn(
+          duration: Duration(
+            seconds: MovieDimensions.posterZoomInDuration,
+          ),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              MovieDetailPoster(
+                moviePosterPath: result.posterPath.toString(),
```

Comment:
Why you parse this to String, posterPath is already a String

Reply:
Corrected.